components/transport-jms/src/test/java/org/wso2/transport/jms/test/client/polling/QueueClientPollingTestCase.java

```diff
@@ -95,7 +95,7 @@ public void queuePollingTestCase()
 
         int receivedMsgCount = 0;
         for (int i = 0; i < 10; i++) {
-            Message message = jmsClientConnectorQueue.poll(queueName, 1000);
+            Message message = jmsClientConnectorQueue.poll(queueName, 1000, null);
```

Comment:
Can we have this value(1000) as a constant?

Reply:
updated :)